src/browser/api/application.js

```diff
@@ -674,7 +675,7 @@ function run(identity, mainWindowOpts, userAppConfigArgs) {
     ofEvents.once(route.window('closed', uuid, uuid), () => {
         delete fetchingIcon[uuid];
         removeTrayIcon(app);
-
+        releaseUuid(uuid);
```

Comment:
This will fire if the app crashes as well right?

Reply:
When the user closes the crashed window it does